lib/pbench/server/api/resources/users_api.py

```diff
@@ -220,9 +214,9 @@ def post(self):
         # Log if the request has an authorization header included and user is still trying to create a new login
         # We would still proceed to login and return a new auth token to the user
         if self.auth.token_auth.current_user():
-            self.logger.warning(
+            self.logger.info(
```

Comment:
Not a big deal; but, technically, the user could already be logged in and this wouldn't trigger ... unless they decide to present the previous login token. That makes the message wording potentially slightly misleading.

Reply:
Agreed.

And we need to know what an admin would do with this message.  Why does somebody need to know that they tried to re-login?

Reply:
We definitely do not want to make a query to see if the user is already logged in; that's irrelevant... and logging such an occurrence would be wasteful and confusing. It's perfectly reasonable, and probably common, for a user to have multiple logins; e.g., from the dashboard and from `pbench-generate-token`, or think of a functional test environment where a "test" user runs multiple tests in parallel.

If we want an admin to be able to track logins, we need administrative tools, something we're currently missing. I can see someone wanting to see how many sessions are active, how many users are logged in, and stuff like that. We should have a robust audit logging system that allows an admin to see who logged in (and out) when, as well as to trace the life cycle of all our resources (including users, datasets). None of that is part of this PR, nor should be!

Reply:
Fixed